practice-app/platon_api/rest_api/login/login.py

```diff
@@ -0,0 +1,94 @@
+"""
+Created on MAY 23, 2020
+This script controls the login functionality api of PLATON_API, using django&mysql backend.
+Endpoint description:
+    http://localhost:8000/api/login/
+    
+    'POST':
+        Produces error
+    'GET':
+        Gets dictionary from body.
+        JSON Format : {  
+                        e_mail = "Your email will be here.",
+                        password = "Your password will be here, hashed for protection." 
+                        token = "Your token will be here!!",
```

Comment:
* I can not understand why the token is an input for your endpoint. Because I think your endpoint will create a token for the person who is currently logged out of the system. So s/he will not have a valid token. Am I right?

Reply:
You are right. I will change it immediately.